pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[project]
+name = "xcp-ng-tests"
+version = "0.1.0"
+description = "Add your description here"
```

Comment:
Are we forced to set a dummy `version` here?

`description` could benefit from a small enhancement :)

Reply:
yes, unfortunately, for the `version`

~~~
error: Failed to parse: `pyproject.toml`
  Caused by: TOML parse error at line 1, column 1
  |
1 | [project]
  | ^^^^^^^^^
`pyproject.toml` is using the `[project]` table, but the required `project.version` field is neither set nor present in the `project.dynamic` list
~~~

agreed for the description :+1: